### PR TITLE
internal set config methods, improve ChargesEthFee

### PR DESCRIPTION
**Pull request by Jonybang**
#951

### CODE REVIEW

File: contracts/traits/ChargesEthFee.sol [hunk 1]
```diff
@@ -9,65 +9,24 @@
...
 
   // INTERNAL
 
-  function _acceptPayment() internal {
-    require(msg.value == ethFee, "Fee and msg.value not equal");
+  function _acceptPayment(bytes32 _key) internal {
+    address payable feeRegistryPayable = address(uint160(feeRegistry()));
+
+    require(msg.value == IEthFeeRegistry(feeRegistryPayable).ethFeeByKey(_key), "Fee and msg.value not equal");
+    feeRegistryPayable.transfer(msg.value);
```
Comment (chebykin) [inline contracts/traits/ChargesEthFee.sol:30]: If a `feeRegistry` is wrapped with proxy, make sure this transfer can pass through its fallback function.

File: contracts/reputation/AbstractProposalManager.sol [hunk 2]
```diff
@@ -18,7 +18,7 @@ import "../traits/ChargesEthFee.sol";
 
 contract AbstractProposalManager is Initializable, ChargesEthFee {
 
-  uint256 constant VERSION = 2;
+  uint256 public constant VERSION = 2;
```
Comment (chebykin) [inline contracts/reputation/AbstractProposalManager.sol:21]: this increases contract size in KB, I remember there were problems with it

File: contracts/registries/EthFeeRegistry.sol [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+ * @title Eth Fee Registry.
+ * @notice Tracks all fees.
+ */
+contract EthFeeRegistry is IEthFeeRegistry, OwnableAndInitializable {
+
+  uint256 public constant VERSION = 1;
+
+  mapping(bytes32 => uint256) public ethFeeByKey;
+
+  address public feeManager;
```
Comment (chebykin) [inline contracts/registries/EthFeeRegistry.sol:26]: Why don't use ACL roles? Using explicitly set roles along with ACL is confusing.

File: contracts/reputation/AbstractProposalManager.sol (hunk 2 above)
Comment (Jonybang) [reply contracts/reputation/AbstractProposalManager.sol:21]: but it should be public to check it from frontend

File: contracts/registries/EthFeeRegistry.sol (hunk 3 above)
Comment (Jonybang) [reply contracts/registries/EthFeeRegistry.sol:26]: I don't think that we need many fee managers or many fee collectors, so don't need to use ACL

**Pull request merged**